tests/test_parser.py

```diff
@@ -57,7 +61,8 @@ def test_schema_nsmap_without_target_namespace(self):
                 <xs:schema xmlns="http://www/default"
                 xmlns:xs="http://www.w3.org/2001/XMLSchema"></xs:schema>"""
 
-        schema = SchemaParser.from_string(xsd, target_namespace="parent")
+        self.parser.target_namespace = "parent"
+        schema = self.parser.from_string(xsd)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'clazz' in method call](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=4930005)